map/editor/command.py

```diff
@@ -16,6 +16,9 @@ def command(shell: DTShell, args):
         print("------>")
         # ---
         # run start-gui-tools
+        image = "duckietown/dt-gui-tools:daffy-amd64"
+        if "--image" in args:
+            image = args[args.index("--image") + 1]
```

Comment:
Two things: 
- Instead of having a default value for the image, simply do not pass the argument `--image` when one is not given;
- Use argparse to parse the argument --image, not `args[args.index("--image") + 1]`